gk/main.c

```diff
@@ -180,10 +186,18 @@ extract_packet_info(struct rte_mbuf *pkt, struct ipacket *packet)
 	return ret;
 }
 
+static inline uint64_t
+calc_request_expire_at(uint64_t now)
+{
+	return now + REQ_EXPIRATION_SEC * 1000000000000L / picosec_per_cycle;
```

Comment:
Should the constant be `LL` instead of only `L` to guarantee that it's at least 64 bits even on 32-bit platforms?